app/src/main/java/org/oppia/android/app/parser/StringToFractionParser.kt

```diff
@@ -50,6 +50,7 @@ class StringToFractionParser {
     return when {
       !normalized.matches(invalidCharsRegex) -> FractionParsingError.INVALID_CHARS
       normalized.startsWith("/") -> FractionParsingError.INVALID_FORMAT
+      normalized.endsWith("/") -> FractionParsingError.INVALID_FORMAT
```

Comment:
I'm actually thinking this should be a submit-time error (I didn't notice before this was added as a real-time error) since it will be a bit noisy (i.e. it'll always show up as you're typing a fraction, which isn't ideal).

Reply:
Updated